packages/console/src/pages/TenantSettings/tabs/Domains/components/CustomDomain/components/ActiveProcess/components/ActiveStepStatus/index.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from 'classnames';
+
+import Failed from '@/assets/images/failed.svg';
+import Success from '@/assets/images/success.svg';
+import { Ring } from '@/components/Spinner';
+
+import * as styles from './index.module.scss';
+
+export enum StepStatus {
+  Finished = 'finished',
+  Failed = 'failed',
+  Pending = 'pending',
+  Loading = 'loading',
```

Comment:
Align the Capitalization?

Reply:
Updated.

Reply:
Hmm, After inspecting our code base, we don't have a convention about how to set up enum values.
It's good to keep them camel-cased since we can map this value with our CSS class names.